dropwizard-testing/src/main/java/io/dropwizard/testing/common/DropwizardTestResourceConfig.java

```diff
@@ -46,7 +46,7 @@
         register(new JacksonFeature(configuration.mapper));
         register(new HibernateValidationBinder(configuration.validator));
         for (Supplier<?> singleton : configuration.singletons) {
-            register(singleton.get());
+            register(singleton.get().getClass());
```

Comment:
Not sure this will work with `PeopleResourceTest`/`PersonResourceTest`

Reply:
yep, meant to have an abstractbinder here too, but missed from the diff.  still failing with that, so working through it.  pushing updated wip diff now